root/layout.tt

```diff
@@ -10,12 +10,22 @@
     <body>
         [%- React.embed(c, 'layout/components/Header') -%]
 
-        [%- IF c.user.is_editing_disabled -%]
+        [%- IF c.user.is_editing_disabled || c.user.are_notes_disabled -%]
             <div class="banner editing-disabled">
                 <p>
-                    [% l('You’re currently not allowed to edit or vote because an admin has revoked your privileges.
-                          If you haven’t already been contacted about why, please {uri|send us a message}.',
-                        { uri => { href => 'https://metabrainz.org/contact', target => '_blank' } }) %]
+                    [%- IF c.user.is_editing_disabled && c.user.are_notes_disabled -%]
+                        [% l('You’re currently not allowed to edit, vote or leave edit notes on other editors’ edits because an admin has revoked your privileges.
```

Comment:
AFAICT the code currently doesn’t allow you to leave comments on your own edits either (maybe except for an initial one) (the `$editor->id == $self->editor_id ||` isn’t used for the check for the disabled flag)?

I agree with this logic, but that makes this wording seem kind of weird, as it suggests that you would still be able to leave notes on your own edits.

Maybe just
```suggestion
                        [% l('You’re currently not allowed to edit, vote or leave edit notes because an admin has revoked your privileges.
```

Reply:
Duh. I noticed this and fixed the React messages, but I guess I forgot the TT ones. Fixing!